README.rst

```diff
@@ -73,7 +73,7 @@ If you would like the development version instead, the command is:
 
 Once installed, run the server with the following command:
 
-\`\ ``megaqc run``
+ ``megaqc run``
```

Comment:
Hmm the formatting on this is off. It's appearing as a quotation.

Reply:
Yeah - basically a leftover from the md -> rst conversion.

I'll just make a code block.